### PR TITLE
fix(MetadataWidget): fix view for large hierarchies

### PR DESCRIPTION
**Pull request by jusa3**
Large hierarchies should no longer be truncated or run over the edge. Moved problematic inline styles to the correct positions. Added some stories for testing.

### CODE REVIEW

File: src/components/widgets/MetadataWidget/MetadataWidget.tsx [hunk 1]
```diff
@@ -46,28 +46,27 @@ function MetadataWidget(props: MetadataWidgetProps) {
             <EuiFlexGroup direction="column">
               <EuiFlexItem>
                 <EuiFlexGroup>
-                  <EuiFlexItem grow={false}>
+                  <EuiFlexItem grow={false} style={{ maxWidth: 600 }}>
```
Comment (jusa3) [inline src/components/widgets/MetadataWidget/MetadataWidget.tsx:49]: those restrict the width of the MetadataWidget, if the description/title/iri are too large.

File: src/components/widgets/MetadataWidget/TabWidget/TabPresentation.tsx [hunk 2]
```diff
@@ -34,6 +33,7 @@ function TabPresentation(props: TabPresentationProps) {
               {
                 content: (
                   <>
+                    <div style={{ overflow: "auto"}}>
```
Comment (jusa3) [inline src/components/widgets/MetadataWidget/TabWidget/TabPresentation.tsx:36]: this is needed for the scrolling bars to appear

**Review by rombaum — APPROVED** [review]

**Pull request merged**